### DISCUSSION TITLE
Choose a License

### DISCUSSION THREAD

**Discussion by Zabuzard** [Together-Java/TJ-Bot#3] [Q&A] [answered]
Which license should we pick for this repository?

What about:

1. [GPL 3](https://tldrlegal.com/license/gnu-general-public-license-v3-(gpl-3))
2. [MIT](https://tldrlegal.com/license/mit-license)
3. [Apache 2.0](https://tldrlegal.com/license/apache-license-2.0-(apache-2.0))

Other proposals are also welcome!

**Comment by Zabuzard** [comment]
I prefer **GPL 3** due to the copyleft aspect.

**Comment by illuminator3** [comment]
[Unlicense](https://unlicense.org/) so that there are basically no restrictions and anyone can use the source code like they want. This would make sense since we're already an open community, accepting everyone to contribute.

**Comment by Zabuzard** [comment, marked as answer]
Converted to GitHub Issue:
https://github.com/Together-Java/TJ-Bot/issues/5
